Sources/Rownd/framework/Automations/RowndUtils.swift

```diff
@@ -58,4 +59,48 @@ internal class RowndUtils {
             textContainers(in: subview)
         }
     }
+    
+    static func stringifyJsonAny(_ any: Any?) -> String {
+        var string = ""
+        
+        guard let any = any else {
+            return ""
+        }
+        
+        if let dataString = any as? String {
+            string = dataString
+        }
+        else if JSONSerialization.isValidJSONObject(any) {
+            do {
+                let jsonData = try JSONSerialization.data(withJSONObject: any)
+                string = String(decoding: jsonData, as: UTF8.self)
+            } catch {
+                print("Error converting array to JSON: \(error)")
```

Comment:
could be something other than array, right? (valid JSON object)

Reply:
Oh true. Yeah it could be a number

Reply:
Not sure if there's anything else. The JSONSerialization should be able to handle arrays and objects I think